nautobot_circuit_maintenance/models.py

```diff
@@ -22,6 +22,7 @@
 MAX_MAINTENANCE_NAME_LENGTH = 100
 MAX_NOTIFICATION_SENDER_LENGTH = 200
 MAX_NOTIFICATION_SUBJECT_LENGTH = 200
+MAX_NOTIFICATION_TOTAL_LENGTH = 16384
```

Comment:
If we need to impose this limit, then this should be documented in the README as the maximum permissible value for the `raw_notification_size` config setting.